### PR TITLE
Add three sections with semantic HTML tags and style using CSS grid and flex-box

### PR DESCRIPTION
**Pull request by Addis-Belete**
We use a CSS grid for three screen sizes to adjust the layout for mobile tablets and laptops. 
We have added three sections Latest news, Categories, and Latest deals.

**Latest News on Tablet View**

latest-news

**Categories on Tablet View**

categories

**Latest Deals on Tablet View**

deals-tablet

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -98,9 +98,77 @@ <h2><a href="#" class="opal">This mac app authomatically detects your duplicate
     </div>
 
 </div>
-    
-     
-    
+<h4 class="latest-header">Latest News ></h4>
+    <div class="latest-news">
```
Comment (addod19) [inline index.html]: Kindly replace `<div class="latest-news">` to `<section class="latest-news">` so we improve the use of HTML semantics. Kindly do this for the others

File: index.html [hunk 2]
```diff
@@ -98,9 +98,77 @@ <h2><a href="#" class="opal">This mac app authomatically detects your duplicate
...
+
+
+<div class="car">
```
Comment (addod19) [inline index.html]: Kindly replace `<div class="car">` to `<article class="car">` so we improve the use of HTML semantics. Kindly do this for the others

File: index.html [hunk 3]
```diff
@@ -98,9 +98,77 @@ <h2><a href="#" class="opal">This mac app authomatically detects your duplicate
...
+<img src="https://lh3.googleusercontent.com/H8wc4zpiBOojdb00UBQL0M8llf9R4-F0zLa5uiPaAup9XbGsEWLaz1V3EA3GTioKA6Vl=s151"
```
Comment (addod19) [inline index.html]: Kindly wrap the `` tag with `<a>` tag

File: index.html [hunk 4]
```diff
@@ -98,9 +98,77 @@ <h2><a href="#" class="opal">This mac app authomatically detects your duplicate
...
+ alt="hydrogen">
+<div class="drive">
+<a href="#" class="hydrogen">Want to drive a hydrogen EV in the US? Move to California</a>
```
Comment (addod19) [inline index.html]: Kindly wrap the title text with the `<h*>` tags before wrapping the `<h*>` tag with the `<a>` tag. Both the image and the title must be a link

File: index.html [hunk 5]
```diff
@@ -183,7 +251,267 @@ <h4 class="ring"><a href="#" class="over">Wolt</a></h4>
 </div>
 
 <br>
+<div class="Categories">
```
Comment (addod19) [inline index.html]: Kindly replace `<div class="Categories">` to `<section class="Categories">`, so we improve the use of HTML semantics. Kindly do this for the others as well

File: index.html [hunk 6]
```diff
@@ -183,7 +251,267 @@ <h4 class="ring"><a href="#" class="over">Wolt</a></h4>
...
+<div class="hard-fork">
+<div >
+<h4><a href="#" class="categories-title">Hard Fork ></a></h4>
+</div>
+<div class="categories-white">
```
Comment (addod19) [inline index.html]: Kindly replace `<div class="categories-white">` with `<article>` tag to improve the use of HTML semantics. Do this for the others

File: index.html [hunk 7]
```diff
@@ -183,7 +251,267 @@ <h4 class="ring"><a href="#" class="over">Wolt</a></h4>
...
+<ul class="categories-list">
+<li><a href="#">Games to play on date night: Rule the galaxy together in Stellaris</a></li>
+<li><a href="#">We need to make remote meetings more boring in 2021</a></li>
+<li><a href="#">The podcasts that got us through 2020</a></li>
+<li><a href="#">How Netflix shapes mainstream culture, explained by data</a></li>
+<li><a href="#">Christmas is better with Tom Vasel’s YouTube board game reviews</a></li>
+</ul>
+</div>
+</div>
+<div class="latest-deals">
```
Comment (addod19) [inline index.html]: Kindly replace `<div class="latest-deals">` with `<section>` tag so we improve the use of HTML semantics. Do this for the others as well

File: index.html [hunk 8]
```diff
@@ -183,7 +251,267 @@ <h4 class="ring"><a href="#" class="over">Wolt</a></h4>
...
+    <div class="latest-deal-top">
+        <h3>
+            <a href="#" class="top-deal-link">Latest deals <i class="fas fa-chevron-right"></i></a>
+        </h3>
+        <a href="" class="top-right-deal">See all deals</a>
+    </div>
+    <div class="latest-deals-container">
+        <div class="latest-deal-item">
```
Comment (addod19) [inline index.html]: Kindly replace `<div class="latest-deal-item>` with `<article>` tag so we improve the use of HTML semantics. Do this for the others :smile:

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, great job on this project :+1: The live link works perfectly  :muscle: The README is looking great :slightly_smiling_face: Most of the requirements have been met :slightly_smiling_face: Let us kindly address the comments below:

- Let us improve the use of HTML semantics, we should use the `<section>` tag for each section, each child of the section should be `<article>` tag
- Kindly avoid repeating code, some of our CSS property values have been repeated, for example; `display: flex` appears 8+ and we should try and clean the code, also check for the other property values
- The PR could be improved to reflect what we specifically did, like name the sections created and the type of positioning used, like grid or flex in the PR would be a great addition
- `Mobile view of latest deals` section is not correctly aligned as in the original. Kindly check and fix it. Kindly check the screenshot below
`Original`
sc1
`Clone`
sc
- `Tablet mode of the latest deals` section is not correctly positioned as in the original. We should have 3 columns but we currently have 4 columns. Let us fix it
`Original`
gn
`Clone`
gn1

Once these are fixed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Addis-Belete** [comment]
Hello @addod19 :wave: 
Thank you for your clear feedback.
We have used the semantic HTML tag section to group the articles and we have worked on the proper mobile view for the latest deal section. We have used CSS classes to group similar repeated code and have greatly reduced the lines of code.
latsest-deals

File: style.css [hunk 9]
```diff
@@ -339,15 +328,331 @@ a {
...
+  font-size: 0.9rem;
+  margin-right: 0.3rem;
+}
+
+.man-icon {
+  display: none;
+}
+
+/* TABLET */
+@media only screen and (min-width: 481px) {
```
Comment (addod19) [inline style.css]: Hi guys, we are supposed to implement a mobile-first approach which is to have only 2 media queries which are for a tablet (768px) and desktop(1024px). There should be no media query for mobile device :computer:

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, congratulations on fixing the comments from the previous review :muscle: However, in fixing one of the comments we introduced a new approach which is not the recommended one. We are to implement a mobile-first approach which means we have 2 media queries for (768px tablets and (1024px) desktop.

Let us kindly fix this and submit another code review :computer: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Addis-Belete** [comment]
Hello @addod19
Thank you for your feedback!
We have used the exact 2 media queries as requested for (768px tablets and (1024px) desktop.

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations team :tada: 

**Thank you for implementing the changes highlighted by the previous reviewer :handshake:** 

**This project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

**Comment(s)**
- In your subsequent PRs, your PR summary should be a brief description of what you did in the milestone. Therefore, in my opinion, you do not have to add your project screenshots: adding screenshots is ideal for the Readme.

**Again, congratulations,** you can now merge to the master branch.
Do not forget to fill the project completion form.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**